crates/ef-testing/src/evm_sequencer/constants.rs

```diff
@@ -59,7 +59,7 @@ lazy_static! {
         ))
         .unwrap()
     );
-    pub static ref FEE_TOKEN_ADDRESS: ContractAddress = ContractAddress(
+    pub static ref ETH_FEE_TOKEN_ADDRESS: ContractAddress = ContractAddress(
```

Comment:
Never asked before but is there no way to construct the ContractAddress type in a simpler manner than going through 3 conversions?

Reply:
There is actually a macro for this, will update